### PR TITLE
 design: Add in XFS filesystem decision 

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: Design.md [hunk 1]
```diff
@@ -21,12 +26,12 @@ Container Linux installer; the core of it will be dd'ing an image to the disk.
 
 The partition layout is still undecided, but initial proposals look something like:
 
-    Number	Type	Purpose
+    Number  Type    Purpose
     -----------------------------------
-    1	fat32	Boot partition/ESP
```
Comment (ajeddeloh) [inline Design.md]: but I like tabs :(

Comment (dustymabe) [reply Design.md]: oh man haha.. do you really want me to change it back?

**Comment by dustymabe** [comment]
fixed up the commit message in the first commit

File: Design.md (hunk 1 above)
Comment (ajeddeloh) [reply Design.md]: I mean kinda. Not trying to start a holy war, but tabs work nicely for tables.

Comment (dustymabe) [reply Design.md]: I fixed it back to what you had before but no matter what I do I can't make it line up correctly in VIM. I set tabstop to 4 and 8 and neither one of them gives me what I would expect.

**Comment by dustymabe** [comment]
merging.. 

really though I would like to know if the columns line up right for you in VIM and what you have `tabstop` set to.

**Pull request merged**